index.js

```diff
@@ -16,10 +17,21 @@ function lint(textEditor) {
 
 	const dir = pkgDir.sync(path.dirname(filePath));
 
+	// no package.json
+	if (!dir) {
+		return [];
+	}
+
 	// ugly hack to workaround ESLint's lack of a `cwd` option
 	const defaultCwd = process.cwd();
 	process.chdir(dir);
 
+	// check if package.json has dependency or devDependency
+	const pkg = loadJson('package.json');
```

Comment:
Set the `cwd` option. I know it works without, but the above hack will be removed eventually.